Cython/Includes/cpython/contextvars.pxd

```diff
@@ -0,0 +1,94 @@
+from cpython.object cimport PyObject
+
+cdef extern from "Python.h":
+
+    ############################################################################
+    # Context Variables Objects
+    ############################################################################
+
+    # PyContext
+    # The C structure used to represent a `contextvars.Context` object.
+
+    # PyContextVar
+    # The C structure used to represent a `contextvars.ContextVar` object.
+
+    # PyContextToken
+    # The C structure used to represent a `contextvars.Token` object.
+
+    # PyTypeObject PyContext_Type
+    # Type object representing the `contextvars.Context` type.
+
+    # PyTypeObject PyContextVar_Type
+    # Type object representing the `contextvars.ContextVar` type.
+
+    # PyTypeObject PyContextToken_Type
+    # Type object representing the `contextvars.Token` type.
+
+    int PyContext_CheckExact(object obj)
+    # Return `true` if `obj` is of type `PyContext_Type`.
+    # `obj` must not be NULL. This function always succeeds.
+
+    int PyContextVar_CheckExact(object obj)
+    # Return `true` if `obj` is of type `PyContextVar_Type`.
+    # `obj` must not be NULL. This function always succeeds.
+
+    int PyContextToken_CheckExact(object obj)
```

Comment:
I would prefer the return type of these to be `bint`, so that Cython can automatically turn them into True/False when the return value needs to be converted to a Python object.